src/main.jsx

```diff
@@ -101,11 +93,12 @@ function cleanProps(props) {
 
 export default class View extends React.Component {
   render() {
-    var 
-      { style, item, box } = this.props,
-      composedStyle = Object.assign({}, item ? mixItemProps(style, this.props) : {}, box ? mixBoxProps(style, this.props) : {}),
-      props = cleanProps(this.props);
-       
+    const { style, item, box } = this.props;
```

Comment:
it's es6 everywhere, so replace `var` with `const`.
+ a little formatting fixes